src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -150,7 +152,34 @@ public ServiceProviderExports([Import(typeof(SVsServiceProvider))] IServiceProvi
         [ExportForVisualStudioProcess]
         public IPackageSettings PackageSettings => GetService<IPackageSettings>();
 
-        T GetService<T>() => (T)serviceProvider.GetService(typeof(T));
+        [ExportForVisualStudioProcess]
+        public ITippingService TippingService
+        {
+            get
+            {
+                var tippingService = GetService<ITippingService>();
+                if (tippingService == null)
+                {
+                    // GetService<TippingService>() was returning null on Visual Studio 2015, so fall back to using new TippingService(...)
```

Comment:
Do you mean `GetService<ITippingService>()`? Also, any idea why this was failing on 2015?

Reply:
Oops, yes I did mean `ITippingService`.

I have no idea why it was failing on Visual Studio 2015, it had me completely stumped. The service appears to be registered correctly (it works on 2017) and it didn't appear to be throwing any exceptions. It's pretty disconcerting that it wasn't working. Maybe I race-condition or Visual Studio bug?

We could register `GitHub.Services.Vssdk` as a MEF assembly and sidestep this issue. I thought it better to leave the workaround as a hint, incase any of the other services start returning null in 2015.

Reply:
Lets leave it like this for the moment 👍